From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
From: Wiz <95232096+GameCubeGBA@users.noreply.github.com>
Date: Sat, 18 Dec 2021 13:14:05 -0500
Subject: [PATCH] 3777: Fix off-by one display error in which checkout progress
 displayed to user starts from 0 instead of 1

Since we are displaying the number of the file in relation to the number of files to be checked out, the progress number should start at 1, instead of 0, because otherwise it gives the impression one more file than there actually are are set to be checked out.
---
 .../projectdata/actions/VersionControlCheckOutAction.java     | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/Ghidra/Framework/Project/src/main/java/ghidra/framework/main/projectdata/actions/VersionControlCheckOutAction.java b/Ghidra/Framework/Project/src/main/java/ghidra/framework/main/projectdata/actions/VersionControlCheckOutAction.java
index ae8aad2cb..04c8bd63e 100644
--- a/Ghidra/Framework/Project/src/main/java/ghidra/framework/main/projectdata/actions/VersionControlCheckOutAction.java
+++ b/Ghidra/Framework/Project/src/main/java/ghidra/framework/main/projectdata/actions/VersionControlCheckOutAction.java
@@ -134,7 +134,7 @@ public class VersionControlCheckOutAction extends VersionControlAction {
 			if (n == 0) {
 				Msg.showError(this, tool.getToolFrame(), "Checkout Failed",
 					"The specified files do not contain any versioned files available for " +
-						"checkeout");
+						"checkout");
 				return false;
 			}
 
@@ -193,7 +193,7 @@ public class VersionControlCheckOutAction extends VersionControlAction {
 				};
 
 				List<DomainFile> failedCheckouts = new ArrayList<>();
-				int progress = 0;
+				int progress = 1;
 				for (DomainFile df : versionedFiles) {
 
 					monitor.checkCanceled();
-- 
2.38.1

